pkgs/development/python-modules/vdirsyncer/default.nix

```diff
@@ -1,47 +1,26 @@
 { lib
-, buildPythonPackage
-, fetchPypi
-, pythonOlder
-, click
-, click-log
-, click-threading
-, requests-toolbelt
-, requests
-, atomicwrites
-, hypothesis
-, pytestCheckHook
-, pytest-subtesthack
-, setuptools-scm
-, aiostream
-, aiohttp-oauthlib
-, aiohttp
-, pytest-asyncio
-, trustme
-, aioresponses
```

Comment:
These separated inputs are supposed to be kept as they were.

Reply:
I reverted my change. What is the reason to prefer listing separated inputs over just using `python3Packages`?

Reply:
That makes overriding individual dependency package easier, also makes it clear what packages are used.